airflow/providers/google/cloud/transfers/gcs_to_gcs.py

```diff
@@ -405,15 +423,25 @@ def _copy_file(self, hook, source_object):
             hook=hook, source_object=source_object, destination_object=destination_object
         )
 
-    def _copy_directory(self, hook, source_objects, prefix):
```

Comment:
Maybe it is better to mark this method as deprecated? since you are changing the name of it and there is a chance that someone could use the old name in his/her code? WDYT?

Reply:
I think it is fine to rename it because it is not a public method, and it is not used anywhere else. It's not supposed to be used outside of the operator.

Reply:
okay :)